test/mongoGetCommand.test.ts

```diff
@@ -958,5 +958,13 @@ suite("scrapbook parsing Tests", () => {
         assert.deepEqual(command.argumentObjects, [{}, {}]);
         assert.deepEqual(command.errors[0].message, "mismatched input 'f' expecting {'{', '[', RegexLiteral, StringLiteral, 'null', BooleanLiteral, NumericLiteral}")
     });
+
+    test("test user issues: https://github.com/Microsoft/vscode-cosmosdb/issues/899 - multi-line comment, not regex", () => {
```

Comment:
If you used testParse it would be a lot easier for you.  Also, it automatically runs with \n changed to \r\n, etc., which I suspect you have a bug with.

Reply:
I looked into using testParse. However, testParse uses ensureSingleCommand. I didn't want to remove it and change all the test cases depending on it, so I created another test case instead.

Reply:
Okay, but then where's the testcase using \r?  We need this since we've had bugs in the past that were Windows only because of this.

Reply:
Thx